homeassistant/components/traccar/device_tracker.py

```diff
@@ -85,6 +87,12 @@
         vol.Optional(CONF_SKIP_ACCURACY_ON, default=[]): vol.All(
             cv.ensure_list, [cv.string]
         ),
+        vol.Optional(CONF_DEVICES_INCLUDE, default=[]): vol.All(
+            cv.ensure_list, [cv.string]
+        ),
+        vol.Optional(CONF_DEVICES_EXCLUDE, default=[]): vol.All(
+            cv.ensure_list, [cv.string]
+        ),
```

Comment:
This PR can not be accepted as is, it will need to be converted to config flow first <https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

> Changes to existing YAML configuration for these same existing integrations will no longer be accepted.

https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md